<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormSerial2</class>
 <widget class="QDialog" name="FormSerial2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>270</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FormSerial2</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: rgb(231, 231, 231);</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>20</y>
     <width>351</width>
     <height>231</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:white;
border:2px dotted black;
border-radius:10px;</string>
   </property>
   <widget class="QComboBox" name="Mac_ComboBox">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>70</y>
      <width>250</width>
      <height>30</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">font:16px &quot;Roboto&quot; ;
background:transparent;
color:black;
border:1px solid black;
border-radius:5px;</string>
    </property>
   </widget>
   <widget class="QLabel" name="Mac">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>60</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font:16px &quot;Roboto&quot;;
background:transparent;
border:0px;</string>
    </property>
    <property name="text">
     <string>MAC</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="titulo">
    <property name="geometry">
     <rect>
      <x>59</x>
      <y>20</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font:16px &quot;Roboto&quot;;
background:transparent;
border:0px;</string>
    </property>
    <property name="text">
     <string>Generador de Seriales</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Serial">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>60</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font:16px &quot;Roboto&quot;;
background:transparent;
border:0px;</string>
    </property>
    <property name="text">
     <string>Serial</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Serial_edit">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>120</y>
      <width>250</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font:16px &quot;Roboto&quot;;
background:transparent;
border:1px solid black;
border-radius:5px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
    </property>
   </widget>
   <widget class="QLabel" name="Dias">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>60</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font:16px &quot;Roboto&quot;;
background:transparent;
border:0px;</string>
    </property>
    <property name="text">
     <string>Dias</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="Days_ComboBox">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>170</y>
      <width>81</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font:16px &quot;Roboto&quot; black;
background:transparent;
color:black;
border:1px solid black;
border-radius:5px;</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
